pallets/polkadex-uniswap/src/lib.rs

```diff
@@ -42,7 +42,7 @@ decl_storage! {
 		/// LPShare holdings
 		LiquidityPoolHoldings get(fn holdings): map hasher(identity) (T::AccountId,(T::Hash,T::Hash)) => FixedU128;
 		/// Swapping Fee
-		SwappingFee: FixedU128 = FixedU128::from_fraction(0.003);
+		SwappingFee: FixedU128 = FixedU128::from(3)/FixedU128::from(1000);
```

Comment:
@Gauthamastro verify this

Reply:
This will work but there might be slight changes in tests.rs because of accuracy.